scripts/kubernetes/updatevalues.js

```diff
@@ -0,0 +1,57 @@
+/*********************************************************************
```

Comment:
Can't have this file here, what calls this? We currently have no code in this repo, but by adding this we will need to take this through SWQRC.  If you took it through with MPS, then its fine, but we'll need that evidence. Otherwise can convert this to a bat file maybe?

Reply:
the launch.bat script calls it to update the values from the .env file to values.yaml. I'll remove it and add it to the docs.